packages/terra-site/webpack.config.js

```diff
@@ -48,11 +48,9 @@ module.exports = {
               ];
             },
           },
-        }, {
+        },
+        {
           loader: 'sass-loader',
-          options: {
-            data: `@import "${path.resolve(path.join(__dirname, 'node_modules/terra-legacy-theme/lib/LegacyTheme.scss'))}"; $terra-bidi: true;`,
```

Comment:
Its been hard coded in the the files that explicitly use the `$terra-bidi` variable. (mixins, datepicker, demographics banner, and slidepanel)